airbyte-integrations/connectors/source-retently/source_retently/schemas/nps.json

```diff
@@ -1,12 +1,35 @@
 {
-    "score": { "type": "integer" },
-    "scoreSum": { "type": "integer" },
-    "metricsType": { "type": "string" },
-    "promoters": { "type": "integer" },
-    "passives": { "type": "integer" },
-    "detractors": { "type": "integer" },
-    "promotersCount": { "type": "integer" },
-    "passivesCount": { "type": "integer" },
-    "detractorsCount": { "type": "integer" },
-    "totalResponses": { "type": "integer" }
+  "type": "object",
+  "properties": {
+    "score": {
+      "type": "integer"
```

Comment:
@henriblancke can you update this to a more common form to not break if you have missing data?
`["null", "integer"]` to all fields.

Reply:
Done updated other schemas as well